blazor-toc.html

```diff
@@ -4155,6 +4155,34 @@
 			</li>
 		</ul>
 	</li>
+	<li>Smart Paste Button
+		<ul>
+			<li>Getting Started
+				<ul>
+					<li> <a href="/blazor/smart-paste/getting-started-webapp">Blazor Web App</a></li>
+					<li>
+						<a href="/blazor/smart-paste/getting-started">Blazor Server App</a>
+					</li>
+				</ul>
+			</li>
+			<li><a href="/blazor/smart-paste/annotation">Annotation</a></li>
+			<li><a href="/blazor/smart-paste/customization">Style and Appearance</a></li>
+		</ul>
+	</li>
+	<li>Smart TextArea
+		<ul>
+			<li>Getting Started
+				<ul>
+					<li> <a href="/blazor/smart-textarea/getting-started-webapp">Blazor Web App</a></li>
+					<li>
+						<a href="/blazor/smart-textarea/getting-started">Blazor Server App</a>
+					</li>
+				</ul>
+			</li>
+			<li><a href="/blazor/smart-textarea/customization">Customize Suggestions</a></li>
+			<li><a href="/blazor/smart-textarea/features">Inherited Features</a></li>
+		</ul>
```

Comment:
If API reference published, Link here

Reply:
API Not yet publihed. Once published will add it